deploy/olm-catalog/konveyor-operator/v1.2.0/konveyor-operator.v1.2.0.clusterserviceversion.yaml

```diff
@@ -374,6 +374,13 @@ spec:
           - apiservers
           verbs:
           - '*'
+        - apiGroups:
```

Comment:
Latest, not v1.2.0 here as well. Here it looks like you edited the clusterPermission (good).